src/pynwb/form/build/map.py

```diff
@@ -215,13 +215,17 @@ def __init__(self, **kwargs):
         self.__carg2spec = dict()
         self.__map_spec(spec)
 
+    def hack_get_subspec_values(self, *args, **kwargs):
```

Comment:
Making a note that we should come back in and rework get_subspec_values.  Ill make an issue for this.